serverTest.py

```diff
@@ -2,24 +2,36 @@
 import server
 
 class TestSocketIO(unittest.TestCase):
-    def test_connect(self):
+    def test_if_connection_successful(self):
+        # arrange & act

```

Comment:
nit: you don't need these comments.  Just make sure that your tests are divided into those (roughly; can be more) 3 sections (new lines in between them is a good way).